naas/models/subscriber_email_address.py

```diff
@@ -35,23 +35,24 @@ def confirmation_code(self):
         """Returns confirmation code for the record"""
         return self.attributes.get('confirmation_code')
 
+    def confirmed_at_value(self):
+        """Returns confirmation date value for the record"""
+        return self.attributes.get('confirmed_at')
+
     def confirmed_at(self):
         """Returns the confirmed at timestamp value"""
         try:
-          return datetime.datetime.strptime(
-            self.attributes.get('confirmed_at'), '%Y-%m-%dT%H:%M:%S%z')
-        except TypeError as e:
+          return iso8601.parse_date(self.confirmed_at_value())
+        except iso8601.iso8601.ParseError as e:
           None
 
     def created_at(self):
         """Returns the created at timestamp value"""
-        return datetime.datetime.strptime(
-            self.attributes.get('created_at'), '%Y-%m-%dT%H:%M:%S%z')
+        return iso8601.parse_date(self.attributes.get('created_at'))
```

Comment:
@arthuroe - any thoughts here on using the [iso8601](https://pypi.org/project/iso8601/) dependency? This seems to be a cleaner abstraction for our use cases. These timestamps will always be returned in that format from the API.

Reply:
Does this change affect what you mentioned below?

> @arthuroe - Almost all of the models will include these timestamps (created_at, updated_at), so anytime we have a Date, Time, or DateTime object we will want to coerce it properly. This way the consumer can use date/time libraries to work with and format the dates.
> 
> This is helpful to:
> 
> Format the dates as they choose
> At the collection level, the client can implement ranges of 'find all records between date X and Y'

Reply:
@arthuroe - The topic is still the same. This dependency wraps up the initial `datetime` calls and eliminates us having to call `strptime` with the string format. In terms of exception handling, the conversation is still the same.

Reply:
Alright. Am okay going forward with `iso8601` dependency.